### PR TITLE
Implement crawlers

### LINKED ISSUE #335: Web Crawlers not working
# Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [x] I am running the latest version
- [x] I checked to make sure that this issue has not already been filed

# Expected Behavior

On running - `scrapy crawl am -o out.csv` 
It should put all data crawled to out.csv file in project folder.

# Current Behavior

Current data output - 
```'data': [{'date': None,
 'image_url': None,
 'newsInDetails': '',
 'news_headline': '\n'
                  '\t  \t\tපරිසරය දූෂණය කරන නිතීවීරෝධී ලී මෝලක් නිසා උකුවෙල '
                  'කැළඹෙයි\t  \t',
 'news_link': '/local-news/item/43055-2019-05-28-05-54-38'},
```
Also it is not able to follow link to next page. 
```
Traceback (most recent call last):
  File "/home/anmol/Desktop/fact-bounty/Crawlers/venv/lib/python3.7/site-packages/scrapy/utils/defer.py", line 102, in iter_errback
    yield next(it)
  File "/home/anmol/Desktop/fact-bounty/Crawlers/venv/lib/python3.7/site-packages/scrapy/spidermiddlewares/offsite.py", line 29, in process_spider_output
    for x in result:
  File "/home/anmol/Desktop/fact-bounty/Crawlers/venv/lib/python3.7/site-packages/scrapy/spidermiddlewares/referer.py", line 339, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "/home/anmol/Desktop/fact-bounty/Crawlers/venv/lib/python3.7/site-packages/scrapy/spidermiddlewares/urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/home/anmol/Desktop/fact-bounty/Crawlers/venv/lib/python3.7/site-packages/scrapy/spidermiddlewares/depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "/home/anmol/Desktop/fact-bounty/Crawlers/news_sites/spiders/am.py", line 40, in parse
    next_link = "http://am.lk" + next_link

```

## Context

A similar problem exists for most of the spiders. Currently, there are 30 spiders in the project and all of them need to be analyzed.

### PR DESCRIPTION
**Pull request by Anmolbansal1**
Restart crawler project with scrapy - 
created a single news item
Remove MongoDB pipeline
Remove all previous crawlers
New features - 
Set elasticsearch pipeline with scrapyelasticsearch package
Add crawlers to all websites

Fixes #335 
Implements #336

### CODE REVIEW

File: Crawlers/news_sites/middlewares.py [hunk 1]
```diff
@@ -20,30 +20,30 @@ def from_crawler(cls, crawler):
...
+    def process_spider_output(self, response, result, spider):
         # Called with the results returned from the Spider, after
         # it has processed the response.
 
         # Must return an iterable of Request, dict or Item objects.
         for i in result:
             yield i
 
-    def process_spider_exception(response, exception, spider):
+    def process_spider_exception(self, response, exception, spider):
```
Comment (ivantha) [inline Crawlers/news_sites/middlewares.py]: @Anmolbansal1 Is there a reason for binding these functions with classes?

File: Crawlers/news_sites/settings.py [hunk 2]
```diff
@@ -5,100 +5,94 @@
...
+ELASTICSEARCH_TYPE = 'story'
 
 # Enable and configure the AutoThrottle extension (disabled by default)
-# See http://doc.scrapy.org/en/latest/topics/autothrottle.html
-# AUTOTHROTTLE_ENABLED = True
+# See https://doc.scrapy.org/en/latest/topics/autothrottle.html
+#AUTOTHROTTLE_ENABLED = True
 # The initial download delay
-# AUTOTHROTTLE_START_DELAY = 5
+#AUTOTHROTTLE_START_DELAY = 5
```
Comment (ivantha) [inline Crawlers/news_sites/settings.py]: We usually keep a space after the #

File: Crawlers/news_sites/middlewares.py (hunk 1 above)
Comment (Anmolbansal1) [reply Crawlers/news_sites/middlewares.py]: This is instantiated by default with `scrapy startproject` command and I think it wouldn't be good to change it

File: Crawlers/news_sites/settings.py (hunk 2 above)
Comment (Anmolbansal1) [reply Crawlers/news_sites/settings.py]: This is instantiated by default with `scrapy startproject` command and I think it wouldn't be good to change it

**Comment by Anmolbansal1** [comment]
I have fixed major of them, please check

**Pull request merged**